presto-docs/src/main/sphinx/release/release-0.259.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.259
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a casting error that sometimes occurred for EXPLAIN queries when the query plan contains LIKE predicates.
+* Add Weibull distribution CDF :func:`weibull_cdf` and inverse CDF :func:`inverse_weibull_cdf` functions.
+* Add support to enable verbose memory exceeded errors by default.
+* Add support for CREATE TYPE syntax.
+
+JDBC Changes
+____________
+* Add method ``getConnectionProperties`` to PrestoConnection to allow for connection properties retrieval after a connection is established. (:pr:`16329`).
+* Add support for partial pushdown of JDBC filters.
+
+Kafka Changes
+_____________
+* Add a new pluggable interface for kafka cluster supplier.
```

Comment:
Kafka should be capitalized.  Also, the interface feels like a bit of an implementation detail. what does it allow me to do as a user/admin?

Reply:
Looks like it adds support for configurable Kafka cluster metadata supplier. This is mainly used to get Kafka cluster metadata such as broker list so that Kafka Connector can communicate with Kafka cluster. Currently it only supports the default mode of  reading metadata data from a static configuration file. 

But from https://github.com/prestodb/presto/pull/16272/files#r675680540 looks like the author did not want include the config parameter in the release notes ?
@beinan  do we want to include this in release notes ?

Reply:
removing this from release note as the author said they will add it later.